.doc_gen/metadata/sqs_metadata.yaml

```diff
@@ -74,10 +74,10 @@ sqs_CreateQueue:
         - sdk_version: 2
           github: gov2/sqs
           excerpts:
-            - description: ''
-              snippet_tags: ['sqs.go-v2.CreateQueue']
+            - description: ""
+              snippet_tags: ["sqs.go-v2.CreateQueue"]
   services:
-    sqs: {CreateQueue}
+    sqs: { CreateQueue }

```

Comment:
The YAML linter will complain about these spaces, so you should configure your IDE to not have them.